CellProfiler_pipelines/README.md

```diff
@@ -0,0 +1,288 @@
+
+    >  Images are typically of higher resolution than is required for granularity measurements, so the default value is 0.25. 
+    For low-resolution images, increase the subsampling fraction; for high-resolution images, decrease the subsampling fraction.
+
+    Since I do not know if the NF1 images are more or less resolution than the ones the Broad Institute was using (as they increased the value to 0.5), I decided it was best to keep the default.
+
+    - "Subsampling factor for background reduction": `Integer value`
+
+    The default is set to `0.5`. Like above, I keep the default value as the documentation states:
+
+    > The subsampling factor for background reduction is usually [0.125 – 0.25]. 
+    This is highly empirical, but a small factor should be used if the structures of interest are large. 
+
+    Since objects within the NF1 images are large, it makes sense to keep the default value.
+
+    - "Radius of the structuring element": `Integer value`
+
+    The default is set to `10`. 
+    I kept the default and what the Broad Institute used as I am unsure how changing the value impacts the results.
+
+    - "Radius of granular spectrum": `Integer value`
+
+    The default is set to `16`. 
+    I kept the default and what the Broad Institute used as I am unsure how changing the value impacts the results.
+
+- **MeasureObjectIntensity:** 
+
+    - No parameters in this module.
+
+- **MeasureImageIntensity:**
+
+    - "Measure the intensity only from areas enclosed by objects?": `Yes` or `No`
+    
+    The default is `No`. It is changed to `Yes` because the goal is it extract features from the objects/cells.
+
+    - "Calculate custom percentiles": `Yes` or `No`
+
+    The default is `No` and I keep this default value due to not knowing how this parameter would help the results.
+
+- **MeasureObjectNeighbors:**
+
+    - "Method to determine neighbors": `Within a specified distance`, `Adjacent`, or `Expand until adjacent`
+
+    The default is `Expand until adjacent`. This default is not used in any module. 
+    This parameter was chosen based on the example used from the Broad Institute.
+
+    If `Within a specified distance`, another parameter is brought up:
+
+    - "Neighbor distance": `Integer value`
+
+    The default is set to `5`. 
+    In one of the module, I increased it to 50 to try and make the measurement more accurate.
+
+    - "Consider objects discarded for touching image border?": `Yes` or `No`
+
+    The default is `Yes`. The default is kept because it is important to consider all the objects within an image. 
+    Excluding them could yield wrong interpretation of the results. 
+    
+    For example, a specific genotype could have more cells clumped together than another genotype.
+    If a lot of those neighbors are not included in the results because they are touching the edges, then you could not see this pattern.
```

Comment:
Are you sure that edge cells are discarded prior to this measurement?

Reply:
Yes because the parameters in the secondary objects modules is set to discard objects touching the edges and this module gives the new nuclei objects that discard any nuclei from cells that touch the edges.